### PR TITLE
Customers API

### PR DESCRIPTION
**Pull request by glendaesutanto**
- `POST` Create customer
https://developers.xendit.co/api-reference/#create-customer

- `GET` Get customer by reference ID
https://developers.xendit.co/api-reference/#get-customer-by-reference-id
---------------------------------
How it is tested
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -897,6 +900,62 @@ QRCode qrCode = QRCode.create(
...
+    String email,
+    String givenNames,
+    String middleName,
+    String surname,
+    String description,
+    String phoneNumber,
+    String nationality,
+    CustomerAddress[] addresses,
+    String dateOfBirth,
+    Map&lt;String, Object&gt; metadata
```
Comment (ErvanAdetya) [inline README.md:925]: new line

Comment (glendaesutanto) [reply README.md:925]: new line gimana ko maksudnyaa? itu udah new line (?)

Comment (ErvanAdetya) [reply README.md:925]:
```suggestion
    Map&lt;String,
    Object&gt; metadata
```

it's supposed to be like this right?

**Review by ErvanAdetya — APPROVED** [review]

File: xendit-java-lib/src/main/java/com/xendit/model/Customer.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+   * @param referenceId Customer reference ID
+   * @return Customer
+   * @throws XenditException XenditException
+   */
+  public static Customer[] getCustomerByReferenceId(String referenceId) throws XenditException {
+    String url = String.format("%s%s%s", Xendit.getUrl(), "/customers?reference_id=", referenceId);
+    return Xendit.requestClient.request(RequestResource.Method.GET, url, null, Customer[].class);
+  }
+
+  public static Customer createCustomerRequest(
```
Comment (stanleynguyen) [inline xendit-java-lib/src/main/java/com/xendit/model/Customer.java:140]: Don't think we normally expose this as `public` but cmiiw

File: xendit-java-library-example/src/main/java/ExampleCreateCustomer.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      );
+      System.out.println(customer.getReferenceId());
+      System.out.println(customer.getEmail());
+      System.out.println(customer.getGivenNames());
+    } catch (XenditException e) {
+      e.printStackTrace();
+    }
+  }
+
+  private static void createCustomer2() {
```
Comment (stanleynguyen) [inline xendit-java-library-example/src/main/java/ExampleCreateCustomer.java:53]: Can we have a more meaningful name for this example?

File: xendit-java-lib/src/main/java/com/xendit/model/Customer.java (hunk 2 above)
Comment (glendaesutanto) [reply xendit-java-lib/src/main/java/com/xendit/model/Customer.java:140]: ah yes I think it must be private

File: xendit-java-library-example/src/main/java/ExampleCreateCustomer.java (hunk 3 above)
Comment (glendaesutanto) [reply xendit-java-library-example/src/main/java/ExampleCreateCustomer.java:53]: got it

**Review by stanleynguyen — APPROVED** [review]

**Review by ErvanAdetya — APPROVED** [review]

**Pull request merged**